### ISSUE TITLE
Parse Problems

### ISSUE THREAD

**Issue by adolfont** [Introduction-to-Functional-Programming/analytic_tableaux#5]
# Example

A sequent such as p, p->q |- q

**Comment by adolfont** [comment]
Some links related to parsing https://www.one-tab.com/page/SiXgf2HSR32oDVhUDvbhbA

**Comment by adolfont** [comment]
Maybe use NimbleParsec https://stefan.lapers.be/posts/elixir-writing-an-expression-parser-with-nimble-parsec/

**Comment by luissimas** [comment]
We'll be dealing with signed formulas in the tableaux, but the input string will be a sequent containing unsigned formulas right?

So given the input: `p, p->q |- q` the parser could produce something like:
```elixir
[
  %{formula: :p, sign: :T},
  %{formula: {:implies, :p, :q}, sign: :T},
  %{formula: :q, sign: :F}
]
```

And those would be the formulas in the root of the proof tree

**Comment by adolfont** [comment]
Exactly, @luissimas. The the input string will be a sequent containing unsigned formulas and a   "|-" sign.

Your example is quite good.
